internal/app/oauth/request.go

```diff
@@ -0,0 +1,116 @@
+package oauth
+
+import (
+	"time"
+
+	"github.com/caos/oidc/pkg/oidc"
+)
+
+var essentialScopes = []string{"openid", "profile", "email"}
+
+type AuthRequest struct {
+	ID            string
+	ClientID      string
+	subject       string
+	code          string
+	state         string
+	ResponseType  oidc.ResponseType
+	scopes        []string
+	RedirectURI   string
+	Nonce         string
+	CodeChallenge *oidc.CodeChallenge
+	createdAt     time.Time
+	authorizedAt  *time.Time
+}
+
+func (a *AuthRequest) GetID() string {
+	return a.ID
+}
+
+func (a *AuthRequest) GetACR() string {
+	return ""
+}
+
+func (a *AuthRequest) GetAMR() []string {
+	return []string{
+		"password",
+	}
+}
+
+func (a *AuthRequest) GetAudience() []string {
+	audiences := []string{
+		appConfig.Domain,
+	}
+
+	if appConfig.Debug {
```

Comment:
```suggestion
	if a.Debug {
```

AuthRequest can have Debug instead of referring a global variable.